server/viame_server/viame_detection.py

```diff
@@ -220,3 +222,41 @@ def save_detection(self, folder, tracks):
         )
 
         return True
+
+    @access.user
+    @autoDescribeRoute(
+        Description("Get detections of a meva clip").modelParam(
+            "folderId",
+            description="folder id of a meva clip",
+            model=Folder,
+            paramType="query",
+            required=False,
+            level=AccessType.READ,
+        )
+    )
+    def get_meva_detection(self, folder):
```

Comment:
This needs to be a part of the `get_detections`  endpoint rather than its own endpoint.  Server should be able to figure out wether there's meva data, viame csv, or json and deserialize appropriately.